### DISCUSSION TITLE
What does Jared even mean?

### DISCUSSION THREAD

**Discussion by cizordj** [jaredpalmer/formik#3634] [Q&A] [answered]
I was reading the documentation about the **useFormik** API and couldn't understand the reasons why someone would use it in the **use cases** section. It says “You are Jared”, naturally I would search on the internet what it means and found this:

![image](https://user-images.githubusercontent.com/32869222/186893056-9f3b9541-70cf-4f68-a5ac-52ce530610a0.png)
https://www.urbandictionary.com/define.php?term=Jared

It that a compliment?

Not everyone on the internet speaks English so can you please be more specific? There was also another thing that took me some time understand:

`- You want to avoid using React Context (possibly for perf reasons)`

Perf means performance right? At first I thought It was a negative thing.

**Comment by SevenOutman** [comment, marked as answer]
Jared is the name of the author of Formik.

The documentation means you should use `useFormik` hook only if it's necessary or you are the author itself who knows well what this hook is doing.
